dt-notifications/notifications.php

```diff
@@ -783,7 +815,7 @@ public static function get_notification_message_html( $notification, $html = tru
             $notification_note = sprintf( esc_html_x( '%1$s subassigned %2$s to you.', 'User1 subassigned contact1 to you.', 'disciple_tools' ), $display_name, $link );
         } elseif ( $notification["notification_name"] ==="milestone" ){
             $meta_value = $notification["field_value"] ?? '';
-            $contact_fields = DT_Posts::get_post_field_settings( "contacts", false ); //no cache to get the labels in the correct language
+            $contact_fields = Disciple_Tools_Contact_Post_Type::instance()->get_custom_fields_settings( true, null, false, false ); //no cache to get the labels in the correct language
```

Comment:
@prykon is no longer valid in v1 :
Disciple_Tools_Contact_Post_Type::instance()->get_custom_fields_settings(

Reply:
@prykon will you be able to update this?

Reply:
@corsacca What is the v1 equivalent of this function? Thanks!

Reply:
What you had there is actually correct.

Reply:
Reverted.